README.md

```diff
@@ -1,85 +1,120 @@
 # Demo deployment of BIG-IPs using Terraform
-Demo deployment of F5 BIG-IP in AWS using Terraform
-
-an authentication token must be generated and recorded as documented below in order to access the modules required by this demo
-https://www.terraform.io/docs/commands/cli-config.html
-
-You can choose to run this from your workstation or a container. Follow the instructions below as appropriate;
-
-# Using your workstation
-- install Terraform https://learn.hashicorp.com/terraform/getting-started/install.html
-- install inpsec https://www.inspec.io/downloads/
-- install locust https://docs.locust.io/en/stable/installation.html
-- install jq https://stedolan.github.io/jq/download/
-- if on a Windows workstation, install Putty for scp support https://putty.org
-
-# Using a Docker container
-The port 8089 is opened in order to use the gui of the locust load generating tool should you choose to use it.
-- install Docker Desktop (https://www.docker.com/products/docker-desktop)
-- `docker run -it -v $(pwd):/workspace -p 8089:8089 mmenger/tfdemoenv:1.6.2 /bin/bash`
-
-# Required Resource
+# Pre-Req
 This example creates the following resources inside of AWS.  Please ensure your IAM user or IAM Role has privileges to create these objects.
 
-**Note:** This example requires 4 Elastic IPs, please ensure your EIP limit on your account can accommodate this (information on ElasticIP limits can be found at https://docs.aws.amazon.com/general/latest/gr/aws_service_limits.html#limits_ec2)
+**Note 1:** This example requires 4 Elastic IPs, please ensure your EIP limit on your account can accommodate this (information on ElasticIP limits can be found at https://docs.aws.amazon.com/general/latest/gr/aws_service_limits.html#limits_ec2)
  - AWS VPC
  - AWS Route Tables
  - AWS Nat Gateways
  - AWS Elastic IPs
  - AWS EC2 Instances
  - AWS Subnets
  - AWS Security Groups
- 
- **Note:** In order to use this demo your AWS account must be subscribed to the F5 AMI and its associated terms and conditions. If your account is not subscribed, the first time ```terraform apply``` is run you will receive an error similar to the following:
 
-```
- Error: Error launching source instance: OptInRequired: In order to use this AWS Marketplace product you need to accept terms and subscribe. To do so please 
+ **Note 2:** In order to use this demo your AWS account must be subscribed to the F5 AMI and its associated terms and conditions. If your account is not subscribed, the first time ```terraform apply``` is run you will receive an error similar to the following:
+
+ **Error:** Error launching source instance: OptInRequired: In order to use this AWS Marketplace product you need to accept terms and subscribe. To do so please 
 visit https://aws.amazon.com/marketplace/pp?sku=XXXXXXXXXXXXXXXXXXXXXXXX
```

Comment:
since the error message reference has been removed, there should be some information about where to find the appropriate version of the sku link.